### ISSUE TITLE
uploading gif

### ISSUE THREAD

**Issue by Dahanael** [GeekMagicClock/smalltv#43]
Hi !
i'm trying to upload gifs on the Model: SmallTV-Ultra,Version: Ultra-V9.0.25. Problem is the memory that is really short, even when i compress gif a lot. isnt there a way to extend the memory by using a local path instead or uploading it directly ? 
(because in my case, i plug the small tv directly into my pc sooo..)
Prob a dumb question, sorry. But i can upload only one small and compressed gif, thats frustrating 
thanks !

**Comment by GeekMagicClock** [comment]
Hi, total free space is about 1MB, if you delete all the gif/jpgs under the weather/photo options, please check.

**Comment by Dahanael** [comment]
Yes, it's still not a lot of memory. Best i Can Do is like 2 gifs
My question was if there wasnt a way to take gifs from a local folder instead of having to upload it ? Because my device is plugged into my pc directly

**Comment by GeekMagicClock** [comment]
No, If you want to stream the GIF file to the device, it's hard to do for this hardware.
